config.md

```diff
@@ -85,7 +85,7 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 
 ## Process configuration
 
-* **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process. Defaults to false.
+* **`terminal`** (bool, optional) specifies whether a terminal is allocated for and attached to that process. Defaults to false.
```

Comment:
I think this is still too vague.  Who allocates the terminal?  It sounds like runC expects the runtime-caller to already have a slave, although ccon uses this setting to have the container process opens the pair with `posix_openpt`.  See also #494.